frontend/packages/model-registry/contract-tests/scripts/run-with-mock-bff.sh

```
@@ -0,0 +1,262 @@
+main() {
+    log_info "🚀 Starting Model Registry contract tests with Mock BFF"
+    
+    check_go
+    # Prune older test runs (keep last 5) to avoid buildup
+    RESULTS_BASE_DIR="$CONTRACT_DIR/contract-test-results"
+    if [[ -d "$RESULTS_BASE_DIR" ]]; then
+        # shellcheck disable=SC2012
+        OLD_RUNS=$(ls -1 "$RESULTS_BASE_DIR" | sort -r | tail -n +6 2>/dev/null || true)
+        for d in $OLD_RUNS; do
+            if [[ -n "$d" ]]; then
+                rm -rf "$RESULTS_BASE_DIR/$d" || true
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)